Client/Frontend/Settings/HomepageSettings/WallpaperSettings/v1/WallpaperSettingsViewModel.swift

```diff
@@ -139,8 +138,18 @@ class WallpaperSettingsViewModel {
 
 private extension WallpaperSettingsViewModel {
 
+    var initialSelectedIndexPath: IndexPath? {
+        for (sectionIndex, collection) in wallpaperCollections.enumerated() {
+            if let rowIndex = collection.wallpapers.firstIndex(where: {$0 == wallpaperManager.currentWallpaper}) {
```

Comment:
```suggestion
            if let rowIndex = collection.wallpapers.firstIndex { $0 == wallpaperManager.currentWallpaper } {
```
nit: we spoke about it.